docs/site/reference.md

```diff
@@ -83,9 +83,13 @@ linkage evaluation phase. The following features are supported:
 
 :   The patient's email address.
 
-`DRIVERS_LICENSE`
+`IDENTIFIER`
 
-:   The patient's driver's license number.
+:   The patient's identifier. This is a catch-all for all identifiers for the patient
+
+`IDENTIFIER:<type>`
+
+:   The patient's specific identifier type. For example, `IDENTIFIER:MRN` would be the patient's medical record number.
```

Comment:
```suggestion
:   The patient's specific identifier type. For example, `IDENTIFIER:MR` would be the patient's medical record number.  Unlike `IDENTIFIER`, this will ONLY compare values of a specific type.  Valid type codes can be found here http://hl7.org/fhir/R4/v2/0203/index.html.
```